test/ecto/changeset_test.exs

```diff
@@ -249,6 +249,10 @@ defmodule Ecto.ChangesetTest do
     assert_raise FunctionClauseError, fn ->
       cast(%Post{}, %{}, %{})
     end
+
+    assert_raise FunctionClauseError, fn ->
+      cast(%Post{}, %{"title" => "foo"}, nil)
+    end
```

Comment:
There already was no way to pass `nil` to `cast/3` so I only added the test to prevent regressions.